test/parallel/test-stream-pipeline.js

```diff
@@ -1406,3 +1406,44 @@ const tsp = require('timers/promises');
   }));
   ac.abort();
 }
+
+{
+  async function run() {
+    let finished = false;
+    let text = '';
+    const write = new Writable({
+      write(data, enc, cb) {
+        text += data;
+        cb();
+      }
+    });
+    write.on('finish', () => {
+      finished = true;
+    });
+
+    await pipelinep([Readable.from('Hello World!'), write]);
+    assert(finished);
+    assert.strictEqual(text, 'Hello World!');
+  }
+
+  run();
```

Comment:
```suggestion
  run().then(common.mustCall());
```

Reply:
https://github.com/nodejs/node/blob/dbc3a05b7d315025d864527da2cddc53d0c3b378/test/parallel/test-stream-pipeline.js#L471

Reply:
You are right, it is not needed.